packages/app-builder-lib/scheme.json

```diff
@@ -5220,6 +5221,22 @@
             "string"
           ]
         },
+        "compression": {
+          "anyOf": [
+            {
+              "enum": [
+                "xz",
+                "lzo"
+              ],
+              "type": "string"
+            },
+            {
+              "type": "null"
+            }
+          ],
+          "default": "xz",
```

Comment:
The default should probably be lzo. There is mostly just one reason for xz: file size. The abysmal decompression speed is absolutely terrible for the kind of applications that are usually built with Electron.

The kde-neon extension wants to enable lzo by default in the future: https://github.com/snapcore/snapcraft/pull/3595

Reply:
@ppd can you propose the change to my PR, I'll merge that in

Reply:
@om26er https://github.com/om26er/electron-builder/pull/1
Contains the changes from the patch above and the lzo default.